test/functionalTest/cases/3190_notify_only_attributes_that_change/notify_only_attributes_that_change_get_and_update_subscription.test

```diff
@@ -0,0 +1,219 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Notify only attributes that change get and update subscription request for onlyChangedAttrs for NGSIv2 POST /v2/subscriptions, GET /v2/subscriptions/{subscription_id}, PATCH /v2/subscriptions/{subscription_id}, GET /v2/subscriptions/{subscription_id}.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+
+--SHELL--
+
+#
+# 01. POST /v2/subscriptions, Create a subscription with onlyChanged: true.
+# 02. GET /v2/subscriptions/{subscription_id}, Get subscription to check onlyChanged: true is rendered
+# 05. PATCH /v2/subscriptions/{subscription_id}, Modify subscription to change onlyChanged to false
+# 06. GET /v2/subscriptions/{subscription_id}, Get subscription to check onlyChanged: false is rendered
```

Comment:
It should be 1, 2, 3, 4 instead of 1, 2, 5, 6

(Only the preamble needs a fix, the echo part seems to be ok)

Reply:
fixed in c089631